api.py

```diff
@@ -36,28 +36,17 @@
 
 oauth2_scheme = OAuth2PasswordBearer(tokenUrl="auth/jwt/login")
 
+@app.post('/email')
```

Comment:
Is the idea to expose the ability to send email in the API for users to use? Why? Can't this be abused for SPAM?

Reply:
You're right, I think we can suppress this line...